modules/apps/adaptive-media/adaptive-media-journal-test/src/testIntegration/java/com/liferay/adaptive/media/journal/internal/exportimport/data/handler/test/AMJournalArticleStagedModelDataHandlerTest.java

```diff
@@ -440,6 +443,24 @@ private String _getStaticContent(FileEntry... fileEntries)
 		return _getContent(sb.toString());
 	}
 
+	private JournalArticle _withPortletImportDisabled(
```

Comment:
I think the test is failing (following the steps on the ticket)

You are now disabling this on the test, so it fails on `if (!ExportImportThreadLocal.isImportInProcess())` 
maybe the test is `_withPortletImportEnabled` (and change to true) but I do not know if this is what you want.